examples/dreambooth/train_dreambooth.py

```diff
@@ -156,7 +156,10 @@ def parse_args(input_args=None):
         ),
     )
     parser.add_argument(
-        "--center_crop", action="store_true", help="Whether to center crop images before resizing to resolution"
+        "--center_crop",
+        default=False,
+        action="store_true",
+        help="Whether to center crop images before resizing to resolution. If not set, images are randomly cropped.",
```

Comment:
Actually, the images are cropped (centre or random) after resizing, not before. I will correct this.